### ISSUE TITLE
[Wasm] Are some controls (FlipView, ScrollViewer) really implemented?

### ISSUE THREAD

**Issue by chris-makaio** [unoplatform/uno#2320]
## Concern?

- [ ] Usage in industry
**- [x] Clarification of capabilities**
- [ ] Getting started with Uno
- [ ] Developing with Uno
- [ ] Contributing to the Uno project
- [ ] Publishing your application
- [ ] Support
- [ ] Other (please specify):

## For which Platform:

- [ ] iOS
- [ ] Android
**- [x] WebAssembly**
- [ ] Windows

## Anything else we need to know?

I'm really new (like, today) to Uno and love the concept. 
I have a lot of UWP background and try to start porting existing apps. One app highly depends on a ScrollViewer for its navigation and I didn't get the ScrollViewer SnapPointTypes to work. I mean, yes, the ScrollViewer is working, but it doesn't snap to its children (= screens of our app).
So I tried using the FlipView control which renders, but doesn't show anything.
It does work in UWP head but doesn't in wasm.

Some sample code to reproduce in playground:
```xaml
<Page
    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
    xmlns:local="using:Uno2"
    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
    mc:Ignorable="d"
    Background="Wheat">


        <FlipView VerticalAlignment="Stretch" MinHeight="400" Background="Blue" x:Name="flipView1" >
           
            <TextBlock FontSize="96">Hello1</TextBlock>
            <TextBlock FontSize="96">Hello2</TextBlock>
            <TextBlock FontSize="96">Hello3</TextBlock>
            <TextBlock FontSize="96">Hello4</TextBlock>
        </FlipView>
   
    
</Page>
```

**Comment by davidjohnoliver** [comment]
No, `FlipView` isn't really implemented on WASM. As you noticed `ScrollViewer` is implemented but the snap-points feature is not. 

Normally controls and members that aren't implemented are marked with the `[NotImplemented]` attribute, but it was missing in the case of `FlipView`, I just created a PR to add it. (The confusion arose because both `FlipView` and snap points were already implemented for Android/iOS when WASM support was added to Uno.)

Actually implementing `FlipView` for WASM is another story and will probably follow from more complete [ListView support](https://github.com/unoplatform/uno/issues/234).
